source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,162 @@
+or a home page split into sections such as introduction, news items and contact information.
 
-When using the `<section>` tag, add an `aria-labelledby` attribute linking to the id of the heading that identifies that section.
+You should add an `aria-labelledby` attribute linking to the id of the heading
+that identifies that section.
 
 Example:
 
-    <section aria-labelledby="advice">
-      <h1 id="advice">Help and advice</h1>
+```html
+<section aria-labelledby="introduction">
+    <h1 id="introduction">Introduction</h1>
+    <p>People have been catching fish for food since before recorded history...</p>
+</section>
 
-## Navigational elements
+<section aria-labelledby="equipment">
+    <h1 id="equipment">Equipment</h1>
+    <p>The first thing you'll need is a fishing rod or pole that you find comfortable
+    and is strong enough for the kind of fish you're expecting to try to land...</p>
+</section>
+```
 
-When using the `<nav>` tag, add the ARIA landmark `role="navigation"`.
+### Navigational elements
 
-The detail of using the `<nav>` tag is covered in this [blog post](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
+Use `<nav role="navigation">` to wrap groups of links that aren't already in
+another context (e.g. `<footer>`). Common examples of navigation sections are
+menus, tables of contents, and indexes.
 
-### Emphasis
+As with `<section>`, it is advisable to use `aria-labelledby` to label your `<nav>`.
+This helps to distinguish different `<nav>` blocks in the page.
 
-If you want to give the current location emphasis within a navigational list, use a `<strong>` tag.
+The GOV.UK blog has [more information regarding the use of the `<nav>` tag](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
 
-Example:
+### Aside
+
+`<aside role="complementary">` should be used for content related to the primary content
+of the webpage, but which does not constitute the primary content of the page.
+
+Examples include author information, related links and related content.
+
+### Footer
+
+`<footer role="contentinfo">` should be used for the footer of the site.
+
+## Individual element guidance
+
+### Headings
+
+Headings should be in order - for example `<h1>` then `<h2>`, but not `<h1>` then `<h3>`.
+
+There should only be one `<h1>` element in a page.
+
+### Text emphasis
+
+Italics should be avoided according to the [GOV.UK content style guide](https://www.gov.uk/guidance/style-guide).
+Bold can be used sparingly, but can make large blocks of text difficult to read.
+
+Semantically, words voiced with emphasis can be marked up with `<em>`, whereas
+words conveying a sense or urgency or warning should be marked up with `<strong>`.
+Browsers traditionally render these in italics and bold respectively, so you may
+need to override `em` to not use italics. In theory, screen readers could adopt
+a different tone of voice for this markup, though none currently do.
```

Comment:
I cannot say for certain that none support this. Maybe saying "none of the major ones currently do" makes that less tenuous. Unless @accessiblewebuk knows more for certain. (He tested this in a couple of screen readers a year or two ago.)
(Not a blocker.)

Reply:
I have tried to find out if there are any which support it. It seems like some of the major ones only support it in a roundabout way (e.g. saying "bold" for `strong` as well as `b` but only on inspection, so not by default). It seems some non-major screen readers might support it (although it's not clear which ones), but they would treat strong/b/em/i all the same.

More info in these articles:
* [Screen Readers lack emphasis](https://developer.paciellogroup.com/blog/2008/02/screen-readers-lack-emphasis/) from 2008
* [Screenreader support for text-level semantics](https://www.brucelawson.co.uk/2018/screenreader-support-for-text-level-semantics/) from 2018
* [Thread: bold vs. strong, italics vs. emphasis](https://webaim.org/discussion/mail_thread?thread=5641) from 2012

Reply:
If we can't be sure of this it might be better to remove this paragraph?

Reply:
Have now changed to "none of the major ones currently do". Agreed that it's worth revisiting later whether or not this is worth advocating for, given the lack of support in screen readers.